collections/editor/observationsubmit.php

```diff
@@ -36,14 +38,14 @@
 	elseif(array_key_exists("CollEditor",$USER_RIGHTS) && in_array($collId,$USER_RIGHTS['CollEditor'])){
 		$isEditor = 1;
 	}
-	if($isEditor && $action == "Submit Observation"){
+	if(($isEditor && $action == "Submit Observation") || $isEditor && $action == "Submit Observation"){
```

Comment:
I don't understand how
`($isEditor && $action == "Submit Observation")`
and
`$isEditor && $action == "Submit Observation"` will be evaluated differently. Is this necessary?